write-fonts/src/tables/glyf/simple.rs

```diff
@@ -58,10 +58,26 @@ impl SimpleGlyph {
     /// * In FontTools endPath says I'm done with this subpath, [BezPath] has no endPath.
     ///
     /// Context courtesy of @anthrotype.
-    pub fn from_kurbo(path: &BezPath) -> Result<Self, BadKurbo> {
-        Ok(simple_glyphs_from_kurbo(std::slice::from_ref(path))?
-            .pop()
-            .unwrap())
+    pub fn from_bezpath(path: &BezPath) -> Result<Self, MalformedPath> {
+        Self::interpolatable_glyphs_from_bezpaths(std::slice::from_ref(path))
+            .map(|mut x| x.pop().unwrap())
+    }
+
+    /// Attempt to create a set of interpolation-compatible glyphs from a set
+    /// of paths.
+    ///
+    /// The paths are expected to be preprocessed, and interpolation compatible
+    /// (i.e. they should have the same number and type of points, in the same
+    /// order.) They should contain only line and quadratic segments; the caller
+    /// is responsible for converting cubics to quadratics  as needed.
+    ///
+    /// This method is provided for use when compiling variable fonts.
+    /// The inputs are expected to be different instances of the same named
+    /// glyph, each corresponding to a different location in the designspace.
```

Comment:
```suggestion
    /// glyph, each corresponding to a different location in the variation
    /// space.
```
This reads a bit better to me as I associate "designspace" with the file format.